### PR TITLE
Add issue templates to the repo

### PR DESCRIPTION
**Pull request by edebar01**
added generic bug report and feature request templates.

also added a template for filing environment logs feedback

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+blank_issues_enabled: false
```
Comment (vizhur) [inline .github/ISSUE_TEMPLATE/config.yaml:1]: I think this one should be updated

File: .github/ISSUE_TEMPLATE/environmentLogsFeedback.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    attributes:
+      label: Things you didn't like
+      description: Please describe what you found to be not helpful or confusing when using the logs.
+    validations:
+      required: true
+  - type: textarea
+    id: suggestions
+    attributes:
+      label: Suggestions
+      description: What suggestions or requests do you have to make the logs more helpful to you?
```
Comment (elesel) [inline .github/ISSUE_TEMPLATE/environmentLogsFeedback.yml:37]:
```suggestion
      description: What would make the logs more helpful to you?
```

File: .github/ISSUE_TEMPLATE/environmentLogsFeedback.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        * For issues with the Azure ML Python SDK, [please open an issue here](https://github.com/azure/azure-sdk-for-python/issues/new/choose)
+  - type: markdown
+    attributes:
+      value: |
+        # Describe your experience
+  - type: textarea
+    id: helpful
+    attributes:
+      label: Things you liked
+      description: Please describe what you found to be helpful when using the logs.
```
Comment (elesel) [inline .github/ISSUE_TEMPLATE/environmentLogsFeedback.yml:23]:
```suggestion
      description: What was most helpful when using this feature?
```

File: .github/ISSUE_TEMPLATE/environmentLogsFeedback.yml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    validations:
+      required: true
+  - type: textarea
+    id: not-helpful
+    attributes:
+      label: Things you didn't like
+      description: Please describe what you found to be not helpful or confusing when using the logs.
```
Comment (elesel) [inline .github/ISSUE_TEMPLATE/environmentLogsFeedback.yml:30]:
```suggestion
      description: What was helpful or confusing when using this feature?
```

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+description: File a bug report for an asset
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for taking the time to fill out this bug report!
+
+        Please note that the bug reports in this repository are for bugs with our community-driven assets.
+
```
Comment (vizhur) [inline .github/ISSUE_TEMPLATE/bug.yml:11]: remove?

File: .github/ISSUE_TEMPLATE/environmentLogsFeedback.yml [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for taking the time to fill out this feedback report!
+
+        Please note that this report is for giving feedback on the environment analysis logs you used to
+        troubleshoot your image build failure.
+
+        * For issues with the ML extension for Azure CLI, [please open an issue here](https://github.com/azure/azure-cli-extensions/issues/new/choose)
```
Comment (vizhur) [inline .github/ISSUE_TEMPLATE/environmentLogsFeedback.yml:13]: can remove references to sdk/cli issues

File: .github/ISSUE_TEMPLATE/config.yaml (hunk 1 above)
Comment (edebar01) [reply .github/ISSUE_TEMPLATE/config.yaml:1]: you mean it should be set to true? so people can create blank issues if they want to

Comment (vizhur) [reply .github/ISSUE_TEMPLATE/config.yaml:1]: I mean config content should be updated to remove CLI/SDK links

Comment (edebar01) [reply .github/ISSUE_TEMPLATE/config.yaml:1]: got it. I removed them

**Review by elesel — APPROVED** [review]

**Pull request merged**